scripts/tabs.js

```diff
@@ -0,0 +1,82 @@
+import jQuery from 'jquery';
+import { jsdom } from 'jsdom';
+
+const $ = jQuery(jsdom().defaultView);
+
+function stripLanguageComment (html) {
+  return html.replace(/(<code [^>]*>)(\/\/ [^\s]*)/, '$1');
+}
+
+function capitalize (languageName) {
+  if (!languageName) {
+    return '';
+  }
+
+  if (languageName === 'php') {
```

Comment:
I saw a duplicate of this code block above. Can it be extracted to some common module?